tests/CLI/modules/firewall_tests.py

```diff
@@ -59,27 +59,32 @@ def test_detail(self):
         result = self.run_command(['firewall', 'detail', 'vlan:1234'])
         self.assert_no_fail(result)
         self.assertEqual(json.loads(result.output),
-                         [{'#': 1,
```

Comment:
You dont need to test the entire output be exact.

My suggestion would be to do something like this:

```
json_result = json.loads(result.output)
self.assertEqual(json_result[0]['action'], 'permit')
```

for whatever fields you want to test for, usually just doing a few spot checks is fine.